packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -132,8 +132,20 @@ class BpkDatepicker extends Component {
       selectionConfiguration,
       minDate,
       maxDate,
+      onClose,
     } = this.props;
-    this.onClose();
+
+    // When the calendar is a single date we always want to close on when a date is selected
+    // or if there is a start date but not yet an end date we don't want the calendar to close until the range is selected.
```

Comment:
This comment looks incorrect. The if statement suggests that we want the the selection to be range and have an end date.

Reply:
Ah yes you are correct since changing the function from original implementation, I missed updating the comment